awx/main/tasks/jobs.py

```diff
@@ -1873,6 +1873,8 @@ def build_args(self, system_job, private_data_dir, passwords):
             if system_job.job_type in ('cleanup_jobs', 'cleanup_activitystream'):
                 if 'days' in json_vars:
                     args.extend(['--days', str(json_vars.get('days', 60))])
+                if 'batch_size' in json_vars:
```

Comment:
we want to move under the

```python
if system_job.job_type == 'cleanup_jobs':
```

block, because `batch-size` is not an argument to the cleanup_activitystream system job